auth-api/email_templates/role_changed_notification_email.html

```diff
@@ -44,14 +44,14 @@
         <tr>
           <td>
             <h1 style="margin: 0; line-height: 34px; letter-spacing: -1px; font-family: Verdana, Arial, sans-serif; font-size: 24px; font-weight: 700;">
-              Your role on the <strong>{{params['org_name']}}</strong> team at Cooperatives Online has been changed.
+              Your role on the <strong>{{params['org_name']}}</strong> account at Cooperatives Online has been changed.
```

Comment:
We'll have to verify this however this wording seems a bit off. Maybe 'Your role for this account...'. I'd check with Claire and Loren on this first.